app/src/main/java/org/openobservatory/ooniprobe/model/settings/Settings.java

```diff
@@ -62,18 +64,19 @@ public Settings(Context c, PreferenceManager pm) {
 		@SerializedName("all_endpoints")
 		public Integer all_endpoints;
 		@SerializedName("randomize_input")
-		public Integer randomize_input;
+		public boolean randomize_input;
 
 		public Options(Context c, PreferenceManager pm) {
-			geoip_asn_path = c.getCacheDir() + "/" + Application.GEO_IPASNUM;
-			geoip_country_path = c.getCacheDir() + "/" + Application.GEO_IP;
-			no_collector = pm.getNoUploadResults();
-			save_real_probe_asn = pm.getIncludeAsn();
-			save_real_probe_cc = pm.getIncludeCc();
-			save_real_probe_ip = pm.getIncludeIp();
+			ca_bundle_path = c.getCacheDir() + "/" + Application.CA_BUNDLE;
+			geoip_asn_path = c.getCacheDir() + "/" + Application.ASN_MMDB;
+			geoip_country_path = c.getCacheDir() + "/" + Application.COUNTRY_MMDB;
+			no_collector = pm.isUploadResults();
+			save_real_probe_asn = pm.isIncludeAsn();
+			save_real_probe_cc = pm.isIncludeCc();
+			save_real_probe_ip = pm.isIncludeIp();
 			software_name = "ooniprobe-android";
 			software_version = VersionUtils.get_software_version();
-			randomize_input = 0;
+			randomize_input = false;
```

Comment:
So, no-one is using `randomize_input`. I'd like to get rid of it in MK. It sounds like extra complexity now.